droopler.install

```diff
@@ -92,3 +94,33 @@ function droopler_install_tasks() {
 
   return $tasks;
 }
+
+/**
+ * Import a config file.
+ *
+ * @param $module
+ *  Module name.
+ * @param $name
+ *  Config file name without .yml extension.
+ */
+function _droopler_import_config($module, $name) {
```

Comment:
Pytanie czy ta funkcja nie powinna być w pliku .module?

Reply:
Na razie ją tu zostawię, bo system updatów jest w trakcie dużego refactoringu - zostanie prawdopodobnie zastąpiona przez klase managera.

Reply:
To dodajmy @todo by nie zapomnieć o tym :)